pages/docs/manual/latest/overview.mdx

```diff
@@ -17,10 +17,12 @@ canonical: "/docs/manual/latest/overview"
 
 ### Comments
 
-| JavaScript        | ReScript    |
-| ----------------- | ----------- |
-| `/* Comment */`   | Same        |
-| `// Line comment` | Same        |
+| JavaScript           | ReScript                         |
+| -------------------- | -------------------------------- |
+| `// Line comment`    | Same                             |
+| `/* Comment */`      | Same                             |
+| `/** Doc Comment */` | `/** Doc Comment Attached */`    |
```

Comment:
Might be worth clarifying what "attached" means. It reads like it's different to JS (since it's not marked as "Same" in the ReScript column).

Reply:
Better now?